build.sh

```
@@ -0,0 +1,2 @@
+sudo docker build -t giahungtechnology/chebichat-nextchat:latest .
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)